core/moduleUtils.ml

```diff
@@ -174,13 +174,25 @@ let create_module_info_map program =
          (get_pattern_variables pat) @ get_binding_names bs
       | {node = Fun (bndr, _, _, _, _); _} :: bs ->
          Binder.to_name bndr :: (get_binding_names bs)
+      | {node = Mutual ms ; _ } :: bs ->
+          get_binding_names ms @ get_binding_names bs
       | _ :: bs -> get_binding_names bs in (* Other binding types are uninteresting for this pass *)
 
     (* Getting type names -- we're interested in typename decls *)
     let rec get_type_names = function
       | [] -> []
-      | { node = Type (n, _, _); _} :: bs -> n :: (get_type_names bs)
-      | _ :: bs -> get_type_names bs in
+      | b :: bs ->
+          begin
```

Comment:
Out of curiosity, why do you use `begin ... end` block here? Is it a matter of style or is there a different reason?

Reply:
It turned out to be unnecessary -- good spot.

Reply:
For the record, there were several such `begin .. end` blocks in the patch that I originally reviewed but I only flagged this one.

Reply:
Ah, I see. I tend to use them whenever I do nested matches (although this one was a `function` and a `match`) to avoid errors due to ambiguity. I'm generally rather conservative with adding them in as the resulting errors can be confusing to work out.